Tests/UnitTests/Purchasing/Purchases/PurchasesLogInTests.swift

```diff
@@ -102,7 +102,7 @@ class PurchasesLogInTests: BasePurchasesTests {
 
     #if ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
 
```

Comment:
not sure how to run these locally

Reply:
We don't have a way right now :/
Before this, it was simply delegating to identity manager, but now it is worth testing.

We should set up a separate target that runs tests with this flag. Mind creating a Linear for that?

Reply:
on it

Reply:
https://linear.app/revenuecat/issue/SDK-3090/create-a-new-test-target-to-test-logic-thats-specific-to

I think a good alternative though would be to just move the logic out of this class and either into PurchasesOrchestrator or a new class, then call it from here. That way we keep this class as just the public Facade, which is what it's supposed to be

Reply:
Totally 👍🏻 then continue testing this with the runtime flag instead.